msal-auth-node/README.md

```diff
@@ -0,0 +1,115 @@
+<!-- Keeping yaml frontmatter commented out for now
+---
+# Metadata required by https://docs.microsoft.com/samples/browse/
+# Metadata properties: https://review.docs.microsoft.com/help/contribute/samples/process/onboarding?branch=main#add-metadata-to-readme
+languages:
+- Node.js
+page_type: sample
+name: "Node.js application that makes a request to a protected API"
```

Comment:
This doesn't call a protected API, does it?

Reply:
Correct! Updating...